linguee/__init__.py

```diff
@@ -74,29 +74,28 @@ def get_suggestions(query):
             result["translations"].append(transl_str)
         results.append(result)
 
-    print(results)
+    # print(results)
```

Comment:
There probably shouldn't be commented code in the final version.

Reply:
Why. Rather completely removed